src/SprykerEco/Zed/Payone/Business/Payment/MethodMapper/CreditCardPseudo.php

```diff
@@ -95,10 +95,10 @@ protected function mapPaymentToAbstractAuthorization(SpyPaymentPayone $paymentEn
         $authorizationContainer->setCurrency($this->getStandardParameter()->getCurrency());
         $authorizationContainer->setPaymentMethod($this->createPaymentMethodContainerFromPayment($paymentEntity));
 
-        $billingAddressEntity = $paymentEntity->getSpySalesOrder()->getBillingAddress();
+        $orderEntity = $paymentEntity->getSpySalesOrder();
```

Comment:
This is bad practice to work with DB entities. Please try to work only with transfer objects

Reply:
Also, it looks like orderEntity is used only by `mapBillingAddressToPersonalContainer` function.
In this case it's suggested to pass `paymentEntity` into function, and let it decide which data is required to make it work.